eclair-core/src/main/scala/fr/acinq/eclair/channel/Commitments.scala

```diff
@@ -179,11 +180,49 @@ case class Commitments(channelId: ByteVector32,
    * and our HTLC success in case of a force-close.
    */
   def almostTimedOutIncomingHtlcs(blockheight: Long, fulfillSafety: CltvExpiryDelta): Set[UpdateAddHtlc] = {
-    def nearlyExpired(add: UpdateAddHtlc) = blockheight >= (add.cltvExpiry - fulfillSafety).toLong
+    def nearlyExpired(add: UpdateAddHtlc): Boolean = blockheight >= (add.cltvExpiry - fulfillSafety).toLong
 
     localCommit.spec.htlcs.collect(incoming).filter(nearlyExpired)
   }
 
+  /** Compute our dust exposure in our commit tx (local) and their commit tx (remote). */
+  def currentDustExposure(): (MilliSatoshi, MilliSatoshi) = {
+    val localCommitDustExposure = CommitmentSpec.dustExposure(localCommit.spec, localParams.dustLimit, commitmentFormat)
+    val remoteCommitDustExposure = CommitmentSpec.dustExposure(remoteCommit.spec, remoteParams.dustLimit, commitmentFormat)
+    (localCommitDustExposure, remoteCommitDustExposure)
+  }
+
+  /** Test whether the given incoming htlc contributes to the dust exposure in the local or remote commit tx. */
```

Comment:
```suggestion
  /** Test whether the given outgoing htlc contributes to the dust exposure in the local or remote commit tx. */
```

Reply:
Done in https://github.com/ACINQ/eclair/pull/1985/commits/d0ffd73c1c1c49ceb1fd7c1b789fbfa9067da98f